service/pom.xml

```diff
@@ -319,6 +319,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.tez</groupId>
+      <artifactId>tez-mapreduce</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Why is this required?

Reply:
otherwise test will fail with MRInput class not found or something similar

Reply:
checked, now all the affected unit tests seem to pass without this dependency, so removing from this PR (maybe a UT which needed this was finally reverted to MR due to some issues and will be taken care of by one of the followup tickets)